### DISCUSSION TITLE
zora explorer error

### DISCUSSION THREAD

**Discussion by drifter728** [blockscout/blockscout#8596] [General]
the token balance ive in zora isnt updated in zora explorer,but i can view my nfts in official zora site

**Comment by Mojmir-R** [comment]
Hey! In such cases, I would advise contacting the Zora team directly as their Blockscout instance is managed by them/Conduit and not by the core Blockscout team. 
In any case, feel free to share more details about this issue here and we are happy to take a look and escalate it to Zora team if needed. 

Thank you.
